Source/Widgets/CesiumInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -0,0 +1,701 @@
+                    } else {
+                        that._eventHandler.removeInputAction(ScreenSpaceEventType.LEFT_CLICK);
+                    }
+                }
+            },
+            _styleString: {
+                default: '{}'
+            },
+            _editorError: {
+                default: ''
+            },
+            _properties: {
+                default: {}
+            }
+        });
+        for (var name in tilesetOptions) {
+            if (tilesetOptions.hasOwnProperty(name)) {
+                // force an update on all options so default event listeners are created
+                knockout.getObservable(that, name).valueHasMutated();
+            }
+        }
+
+        this.propertiesText = knockout.pureComputed(function() {
+            var names = [];
+            for (var prop in that._properties) {
+                if (that._properties.hasOwnProperty(prop)) {
+                    names.push(prop);
+                }
+            }
+            return names.join(', ');
+        });
+    }
+
+    defineProperties(Cesium3DTilesInspectorViewModel.prototype, {
+        /**
+         * Gets or sets the tileset of the view model
+         * @memberof Cesium3DTilesInspectorViewModel.prototype
+         *
+         * @type {Cesium3DTileset}
+         */
+        tileset: {
+            get: function() {
+                return this._tileset;
+            },
+
+            set: function(tileset) {
+                this._feature = undefined;
+                this._tilesetLoaded = when.defer();
+                this._tileset = tileset;
+            }
+        },
+
+        /**
+         * Gets the current feature of the view model
+         * @memberof Cesium3DTilesInspectorViewModel.prototype
+         *
+         * @type {Cesium3DTileFeature}
+         */
+        feature: {
+            get: function() {
+                return this._feature;
+            }
+        },
+
+        /**
+         * Gets the command to trim tile cache
+         * @memberof Cesium3DTilesInspectorViewModel.prototype
+         *
+         * @type {Command}
+         */
+        trimTilesCache: {
+            get: function() {
+                var that = this;
+                return createCommand(function() {
+                    if (defined(that._tileset)) {
+                        that._tileset.trimLoadedTiles();
+                    }
+                });
+            }
+        },
+
+        _compileStyle: {
+            get: function() {
+                var that = this;
+                return createCommand(function() {
+                    that.styleString = that._styleString;
+                });
+            }
+        },
+
+        _checkCompile: {
+            get: function() {
+                var that = this;
+                return createCommand(function(sender, event) {
+                    if (event.ctrlKey && event.key === "Enter") {
```

Comment:
Ctrl+Enter doesn't work for me still, event.key is "↵". Should this be `event.code`?